### ISSUE TITLE
PollNextEvent not polling button events anymore since update

### ISSUE THREAD

**Issue by mat-ieu** [ValveSoftware/openvr#848]
Since the last (automatic) update of SteamVR calls to PollNextEvent(...) don't generate button-related events anymore, like eventvr::VREvent_ButtonPress, VREvent_ButtonTouch.

I also updated the OpenVR headers, but this didn't help.

However, hellovr_opengl still reacts to the m_actionHideCubes button, generated using SetActionManifestPath() and GetActionHandle().

Any help is highly appreciated.

**Comment by JoeLudwig** [comment]
When I run "vrcmd --showevents", I see all those events for all the buttons. The HelloVR sample in the latest SDK won't get those events because it uses the new input system (which doesn't use the events) but any legacy application should.

Are you calling SetActionManifestPath in your application? That would indicate you intend to use the new system and turn off legacy input.

**Comment by mat-ieu** [comment]
Thank you for your answer. With the vrcmd command I can also see the events.

In my programm I got these events at startup, after I removed SetActionManifestPath():
1703 ActionManifestReloaded 
1701 BindingLoadFailed
1702 BindingLoadSuccessful
[...]

I had to rename my exe file in order to ignore an old setting of ActionManifestPath. Now I get these initial events, and my buttons work again!
1703 ActionManifestReloaded 
1702 BindingLoadSuccessful
1702 BindingLoadSuccessful
[...]

By the way: Just adding "2" to the exe did not help to forget the old setting of the ActionManifestPath.
